### PR TITLE
Show thread name on thread listing page

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/4155

## Description of Changes
- Adds the thread topic on the listing page

## Test Plan
- Ensure the thread topic on the listing page

## Deployment Plan
N/A

## Other Considerations
N/A

### Screenshots
image

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/discussions/recent_threads_header.tsx [hunk 1]
```diff
@@ -153,49 +153,49 @@ export const RecentThreadsHeader = ({
...
+              }}
+              disabled={!app.user.activeAccount}
+            />
+          )}
+        </div>
+      </div>
+
+      {isUndefined(topic) && (
+        <CWText className="subheader-text">
+          This section is for the community to discuss how to manage the
```
Comment (zakhap) [inline packages/commonwealth/client/scripts/views/pages/discussions/recent_threads_header.tsx:193]: When the topic is not undefined, we should show the Topic's Description here too. 

When "All Discussions" (topic undefined), this description here should be changed. I will work on some UX copy for this imminently so we can fix this within the PR.

**Review by zakhap — CHANGES_REQUESTED** [review]

File: packages/commonwealth/client/scripts/views/pages/discussions/recent_threads_header.tsx (hunk 1 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/pages/discussions/recent_threads_header.tsx]: [Done](https://github.com/hicommonwealth/commonwealth/pull/4157/commits/19f97f3375aa470d1b95def2effd6b27b4dbd0f6)

Comment (zakhap) [reply packages/commonwealth/client/scripts/views/pages/discussions/recent_threads_header.tsx]: Actually, let's remove default description text altogether for the "All Discussions" page. Only Topics with Descriptions should render a description.

Comment (zakhap) [reply packages/commonwealth/client/scripts/views/pages/discussions/recent_threads_header.tsx]: Otherwise, this PR looks ready to go 👍

Comment (jnaviask) [reply packages/commonwealth/client/scripts/views/pages/discussions/recent_threads_header.tsx]: cc @mzparacha

**Review by jnaviask — APPROVED** [review]

**Pull request merged**